.golangci.yml

```diff
@@ -19,4 +19,4 @@ issues:
   max-same-issues: 0
   exclude:
     # exclude stuttering linting error
-    - ".*and that stutters;.*"
\ No newline at end of file
+    - ".*and that stutters;.*"
```

Comment:
We should add a new line here?

Reply:
Yeah, new line was missing at the end of file, so I added it.

Reply:
Oh! GitHub UI confused me
Looks good!